docs/site/content/docs/tkr-managed-cluster.md

```diff
@@ -0,0 +1,56 @@
+# Deploy Clusters with Different Kubernetes Versions
```

Comment:
These instructions won't work for workload clusters.

Resolving available TKrs is all backed into the `tanzu` CLI and shouldn't be resolved using queries with `imgpkg`.

I'll send a commit to this PR updating the flow.